### DISCUSSION TITLE
Troubleshooting: Ente Auth Password Confirmation Issue and Invalid Recovery Key

### DISCUSSION THREAD

**Discussion by TYTheBeast** [ente-io/ente#3750] [Q&A]
# Troubleshooting: Ente Auth Password Confirmation Issue and Invalid Recovery Key

I'm experiencing two related issues with Ente Auth on my computer:

1. **Password Confirmation Problem:**
   When trying to confirm my password, Ente Auth displays a message stating that my computer is not powerful enough to perform this action. This is preventing me from accessing my account through the desktop application.

2. **Invalid Recovery Key:**
   In an attempt to resolve the password confirmation issue, I tried using my recovery key. However, Ente Auth claims that the recovery key is incorrect, even though I've directly copy-pasted it from the Ente Auth app on my phone.

## Additional Information:

- Ente Auth version on desktop: 4.0.2-1
- Ente Auth version on mobile: 4.0.3
- Desktop operating system and version: EndeavourOS Linux, rolling

## Troubleshooting Steps Taken:

- Deleted `~/.local/share/io.ente.auth/`

Any assistance in resolving these issues would be greatly appreciated. Thank you in advance for your help!

**Comment by fo-od** [comment]
im having the same issue, on arch using version 4.1.6-1 from the AUR

**Reply by fo-od** [reply]
i used ente-auth-bin and it worked
